Exebite.DataAccess/Repositories/CommentRepository/CommentCommandRepository.cs

```diff
@@ -18,9 +19,39 @@ public CommentCommandRepository(IMealOrderingContextFactory factory)
             _factory = factory ?? throw new ArgumentNullException(nameof(factory));
         }
 
+        /// <summary>
+        /// Deletes comment entity
+        /// </summary>
+        /// <param name="commentId">Comment id</param>
+        /// <param name="commentDto">Comment properties</param>
+        /// <returns>Flag that states is entity deleted successfully</returns>
+        public Either<Error, bool> Delete(int commentId, DeleteCommentDto commentDto)
+        {
+            try
+            {
+                using (var context = _factory.Create())
+                {
+                    var comment = context.Comment.SingleOrDefault(c => c.MealId == commentDto.MealId && c.CommentId == commentId && c.CustomerId == commentDto.CustomerId);
```

Comment:
You can also use following `DbContext.Find` method:

```suggestion
                    var comment = context.Comment.Find(mealId, commentId);
```

**Note** that the members of the composite key need to be provided in the proper order to the `DbContext.Find` method.